mysql-test/suite/rocksdb_rpl/r/rpl_rocksdb_row_crash_safe.result

```diff
@@ -195,6 +195,8 @@ END|
 
 
 STOP SLAVE SQL_THREAD;
+Warnings:
```

Comment:
Here and everywhere in this file
Probably it's better to change the command in the .test file

Reply:
This warning comes from extra/rpl_tests/rpl_crash_safe.inc. This is upstream file.
1. Upstream tests that use this file were rerecorded with warning
2. We followed the same path as upstream (rerecorded)
Why?
If we changed as proposed we would need to rerecord upstream tests results as well. In such case when upstream fixes this issue (which I expect) we will have merge conflicts.
So for now let's follow upstream way, when they fix the issue in inc file, we will rerecord again.

Please note that it was changed everywhere as proposed where our tests didn't interfere with upstream files (i.e. STOP SLAVE -> STOP REPLICA in our tests)